service-tests/coveralls.js

```diff
@@ -81,7 +81,7 @@ t.create('show coverage for bitbucket with branch')
   .expectJSON({ name: 'coverage', value: '50%' });
 
 t.create('github coverage')
-  .get('/jekyll/jekyll.json')
+  .get('/github/jekyll/jekyll.json')
```

Comment:
Could you add a test of the old URL (without github) so we can be assured that stays working?

Reply:
sure, good idea :)